File: keycloak-client/src/main/java/cc/coopersoft/accesstoken/keycloak/model/KeycloakTokenInfo.java
package cc.coopersoft.accesstoken.keycloak.model;

import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;

@Data
@NoArgsConstructor
public
class KeycloakTokenInfo{
  private String accessToken;
  private long expiresIn;
  private String refreshToken;
  private long refreshExpiresIn;
  private String tokenType;
  private int notBeforePolicy;
  private String sessionState;
  private String scope;
  private Date created;
}

File: keycloak-client/src/main/java/cc/coopersoft/accesstoken/keycloak/model/KeycloakTokenInfo.kt
package cc.coopersoft.accesstoken.keycloak.model

import java.util.Date

class KeycloakTokenInfo {
    val accessToken: String? = null
    val expiresIn: Long = 0
    val refreshToken: String? = null
    val refreshExpiresIn: Long = 0
    val tokenType: String? = null
    val notBeforePolicy = 0
    val sessionState: String? = null
    val scope: String? = null
    var created: Date? = null
}